google/api_core/bidi.py

```diff
@@ -277,7 +278,11 @@ def open(self):
         request_generator = _RequestQueueGenerator(
             self._request_queue, initial_request=self._initial_request
         )
-        call = self._start_rpc(iter(request_generator), metadata=self._rpc_metadata)
+        if self._rpc_metadata:
+            call = self._start_rpc(iter(request_generator), metadata=self._rpc_metadata)
+        # use metadata from self._start_rpc if no other metadata is specified
+        else:
+            call = self._start_rpc(iter(request_generator))
```

Comment:
When I set up BidiRPC with `LoggingV2Transport().tail_log_entries`, that's the unwrapped method, not the wrapped method. Will this still get the metadata from the wrapped method?

Reply:
also, I tried passing in LoggingServiceV2Client().tail_log_entries and it didn't seem to work.

I'm actually not sure how pubsub passes in the wrapped method successfully.
https://github.com/googleapis/python-pubsub/blob/e907f6e05f59f64a3b08df3304e92ec960997be6/google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py#L524

Reply:
> When I set up BidiRPC with LoggingV2Transport().tail_log_entries, that's the unwrapped method, not the wrapped method. Will this still get the metadata from the wrapped method?

I believe the wrapped one has to be passed in (the wrapper adds the metadata to the call)

> also, I tried passing in LoggingServiceV2Client().tail_log_entries and it didn't seem to work.
>
> I'm actually not sure how pubsub passes in the wrapped method successfully.

Hmm I'll go back and poke at what is actually expected to be passed in to Bidi.